packages/generator/src/entity/namespace.ts

```diff
@@ -31,14 +31,17 @@ function properties(entity: VdmEntity): VariableStatementStructure[] {
 
 function property(prop: VdmProperty, entity: VdmEntity): VariableStatementStructure {
   const type = `'${prop.edmType.startsWith('Edm') ? prop.edmType : prop.edmType.split('.').pop()}'`;
+  const initilizer = prop.isComplex
```

Comment:
`initilizer` is misspelled. Should be `initializer`
```suggestion
  const initializer = prop.isComplex
```